app/views/games/show.html.erb

```diff
@@ -12,8 +12,17 @@
 
     <% 8.times do |col_index|  %>
       <td>
-          #code goes here
+          <% def piece_by_position(game, x, y)
+            piece = game.piece_at(x,y)
+            if piece.nil?
+              return ""
+            else
+            return piece.to_ascii_piece
+            end
+          end %>
```

Comment:
Let's do something different here - you have a method inserted into a view.

How about this:

```
<td>
  <% piece = Piece.find_by_position(x, y) %>
  <% if piece.present? %>
    <%= piece.to_ascii_piece %>
  <% end %>
</td>
```